database/src/env.rs

```diff
@@ -1,20 +1,46 @@
 //! Abstracted database environment; `trait Env`.
 
 //---------------------------------------------------------------------------------------------------- Import
-use std::path::Path;
+#[allow(unused_imports)] // docs
+use crate::ConcreteEnv;
```

Comment:
for docs you can do: `[ConcreteEnv](crate::ConcreteEnv)` instead of having to do this